item.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+
+class Item
+  attr_reader :id, :archived
+  attr_accessor :publish_date
+
+  def initialize(id, publish_date)
+    @id = rand(1..1000)
+    @publish_date = publish_date
+    @archived = false
```

Comment:
we can do the same thing here, the could may look like this: `@archived = archived || false`